mailpoet/lib/WooCommerce/Helper.php

```diff
@@ -78,4 +81,21 @@ public function getRawPrice($price, array $args = []) {
   public function getAllowedCountries(): array {
     return (new \WC_Countries)->get_allowed_countries() ?? [];
   }
+
+  public function wasMailPoetInstalledViaWooCommerceOnboardingWizard(): bool {
+    $wp = ContainerWrapper::getInstance()->get(WPFunctions::class);
```

Comment:
Just noting here that I opted to call ContainerWrapper::getInstance() instead of using dependency injection because the Helper class is instantiated directly in several places and I didn't want to expand too much the scope of this ticket and look into replacing the direct instantiations of the Helper class with DI all over or manually updating the instantiations to add the new dependency.